gui/GameConqueror.py

```diff
@@ -1040,9 +1039,26 @@ def check_backend_version(self):
         if self.backend.get_version() != VERSION:
             self.show_error(_('Version of scanmem mismatched, you may encounter problems. Please make sure you are using the same version of GameConqueror as scanmem.'))
 
-    
 
 if __name__ == '__main__':
+    # Parse cmdline arguments
+    parser = argparse.ArgumentParser(prog='GameConqueror',
+                                     description="A GUI for scanmem, a game hacking tool"
+
+                                    )
+    parser.add_argument('-v', '--version', action='version', version='%(prog)s ' + VERSION)
```

Comment:
I know.
The "best" way would be to slot the `-h` and `-v` handling in the bash script and suppress them in the .py , but I was not willing to look up how to parse cmdline args from bash.

You can put it on a TODO, but IMHO it's still an improvement from what was before.

These cmdline arguments are more for fast testing (`python GameConqueror.py $(pidof geany) -s'123'` is my standard invocation) than really for end users.